update_employees.py

```diff
@@ -70,7 +80,61 @@ def get_employee_data():
     return drop_empty_positions(records_hr_unfiltered)
 
 
+def get_emails_data():
+    """
+    Read csv from shared drive with employee emails
+    :return: dictionary of emails
+    """
+    employee_emails = {}
+
+    smbclient.ClientConfig(username=os.getenv("SHAREDDRIVE_USERNAME"), password=os.getenv("SHAREDDRIVE_PASSWORD"))
+
+    emails_csv = os.getenv("SHAREDDRIVE_FILEPATH")
+    with smbclient.open_file(emails_csv, mode='r') as emails:
```

Comment:
wow—cool!

Reply:
my same reaction when Sergio showed me the python package